### PR TITLE
Removed poll from homepage

### PR DESCRIPTION
**Pull request by alma-castant**
Left commented so it can be used in the next edition if necessary

### CODE REVIEW

**Comment by DMips** [comment]
@alma-castant thanks - i forgot about it ;)

File: app/views/home/index.html.slim [hunk 1]
```diff
@@ -47,20 +47,20 @@
...
-            .form-group
-              .radio
-                label
-                  = f.radio_button :answer, language, checked: language == "C"
-                  = language
-              - if language == "Other"
-                = text_field_tag :other_language, nil, class: "form-control"
-          .form-group
-            = f.submit "Submit", class: "btn btn-pink"
+/!    .page-header
```
Comment (DMips) [inline app/views/home/index.html.slim]: it would be better to extract it to a partial instead of commenting it out

**Comment by alma-castant** [comment]
Updated

**Comment by DMips** [comment]
:+1:

**Pull request merged**